tools/donate-cpu.py

```diff
@@ -282,7 +282,7 @@ def scan_package(work_path, cppcheck_path, jobs):
     cmd = 'nice ' + cppcheck_cmd
     returncode, stdout, stderr, elapsed_time = run_command(cmd)
     print('cppcheck finished with ' + str(returncode))
-    if returncode == -11 or stderr.find('Internal error: Child process crashed with signal 11 [cppcheckError]') > 0:
+    if returncode == -11 or stderr.find('Internal error: Child process crashed with signal 11 [cppcheckError]') > 0 or returncode == -6 or stderr.find('Internal error: Child process crashed with signal 6 [cppcheckError]') > 0:
```

Comment:
hmm what do you think about this instead:

    if returncode in (-11,-6) or stderr.find('Internal error: Child process crashed with signal '):

Reply:
The first part makes sense, but we don't want to create stack traces for any error. I will shorten the second part as well